mne/beamformer/_compute_beamformer.py

```diff
@@ -396,12 +396,12 @@ def _compute_power(Cm, W, n_orient):
     source_power = np.zeros(n_sources)
     for k in range(n_sources):
         Wk = W[n_orient * k: n_orient * k + n_orient]
-        power = Wk.dot(Cm).dot(Wk.T)  # XXX why no conj on Wk?
```

Comment:
@wmvanvliet I am confused, I corrected all of these before?

Reply:
sorry, git mistake on my end.